R/util.R

```diff
@@ -5,3 +5,17 @@ scalar <- function(x) {
 package_version_string <- function(name) {
   as.character(utils::packageVersion(name))
 }
+
+system_file <- function(...) {
+  tryCatch({
+    system.file(..., mustWork = TRUE, package = "daedalus.api")
+  }, error = function(e) {
+    stop(sprintf("Failed to locate file from args\n%s",
+                 paste(list(...), collapse = " ")))
+  })
+}
+
+read_json <- function(filename) {
+  json <- jsonlite::fromJSON(system_file("json", filename))
+  json
+}
```

Comment:
Ah good point. How about if I change the name to something not shared with jsonlite to make it clear this is a special for the API? `read_local_json_file`? I'm just thinking that neither `fromJSON` nor `read_json` are going to do exactly the same as their jsonlite equivalent since it's selecting the folder to read from.

Reply:
`read_local_json()` sounds good!